### DISCUSSION TITLE
App language based on browser preferences?

### DISCUSSION THREAD

**Discussion by Zulaxy** [community/community#65694] [Discussions] [answered]
### Select Topic Area

Question

### Body

I have an app on React and I was wondering what tool can I use so that I can dynamically change my website's language within the texts based on the browser's preferences?

**Comment by Wickedlolz** [comment, marked as answer]
React Intl is popular library for internationalization (i18n) and localization (l10n) in React applications. It provides a comprehensive set of tools and components for formatting dates, numbers, and strings, as well as handling pluralization and messages in multiple languages.
